llvm/test/CodeGen/AMDGPU/misaligned-vgpr-regsequence.mir

```diff
@@ -0,0 +1,33 @@
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx942 -start-after=si-load-store-opt %s -o - | FileCheck %s
```

Comment:
This is a weird way to write the test. Can you replace this with an end to end IR test that hit this problem

Reply:
I can't add the end-to-end IR as https://github.com/llvm/llvm-project/pull/154115 was reverted which showed the code path towards misaligned `v_mov_b64`. I was planning to add the IR test as a reland of https://github.com/llvm/llvm-project/pull/154115 after this has landed but I can add it as a sort of precommit in here since it's related.